dotCMS/src/main/java/com/dotcms/aspects/interceptors/LogTimeMethodInterceptor.java

```diff
@@ -0,0 +1,41 @@
+package com.dotcms.aspects.interceptors;
+
+import com.dotcms.aspects.DelegateMethodInvocation;
+import com.dotcms.aspects.MethodInterceptor;
+import com.dotcms.repackage.com.google.common.annotations.VisibleForTesting;
+import com.dotcms.util.LogTimeUtil;
+
+import java.lang.reflect.Method;
+
+/**
+ * Method handler for the {@link com.dotcms.util.LogTime} annotation aspect
+ * @author jsanca
+ */
+public class LogTimeMethodInterceptor implements MethodInterceptor<Object> {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a "static final long serialVersionUID" field to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2057)